hack/deploy-hub-local/build-edgecluster.sh

```diff
@@ -65,9 +66,11 @@ echo ">>>> Set the Pull Secret"
 echo ">>>>>>>>>>>>>>>>>>>>>>>>"
 echo $OC_PULL_SECRET | tr -d [:space:] | sed -e 's/^.//' -e 's/.$//' >./openshift_pull.json
 
-echo ">>>> Install SWTPM to enable TPMv2"
-echo ">>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>"
-yum install swtpm
+if [ "$TPM" == "true" ]; then
+  echo ">>>> Install SWTPM to enable TPMv2"
+  echo ">>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>"
+  yum install swtpm
```

Comment:
yum -y install swtpm

Reply:
I will actually remove this if here and apply your suggestion